test/unit/discovery/dns_test.go

```diff
@@ -26,7 +24,7 @@ func TestDns(t *testing.T) {
 // ------
 // tests if is able to get txtData from local DNS server
 func testDns(t *testing.T) {
-	targetTxts := []*discovery.TxtData{
+	/*targetTxts := []*discovery.TxtData{
```

Comment:
Can these tests be directly removed since they refer to the old type? (to respect the "no commented code" policy)

Reply:
Tests will be completely rewritten in the next pr, this is a reminder for me of which kind of test we were doing